scripts/update/jellyfin.sh

```diff
@@ -1,87 +1,87 @@
+        [[ -d "/home/${username}/.config/Jellyfin/root" ]] && cp -fRT "/home/${username}/.config/Jellyfin/root" "/var/lib/jellyfin/root" | :
+        #
+        sed -r 's#<PublicPort>(.*)</PublicPort>#<PublicPort>8096</PublicPort>#g' -i "/etc/jellyfin/system.xml"
+        sed -r 's#<PublicHttpsPort>(.*)</PublicHttpsPort>#<PublicHttpsPort>8920</PublicHttpsPort>#g' -i "/etc/jellyfin/system.xml"
+        sed -r 's#<HttpServerPortNumber>(.*)</HttpServerPortNumber>#<HttpServerPortNumber>8096</HttpServerPortNumber>#g' -i "/etc/jellyfin/system.xml"
+        sed -r 's#<HttpsPortNumber>(.*)</HttpsPortNumber>#<HttpsPortNumber>8920</HttpsPortNumber>#g' -i "/etc/jellyfin/system.xml"
+        sed -r 's#<RequireHttps>false</RequireHttps>#<RequireHttps>true</RequireHttps>#g' -i "/etc/jellyfin/system.xml"
+        #
+        if [[ -f /etc/nginx/apps/jellyfin.conf ]]; then
+            sed -r 's#proxy_pass https://127.0.0.1:(.*);#proxy_pass https://127.0.0.1:8920;#g' -i "/etc/nginx/apps/jellyfin.conf"
+            systemctl -q restart "nginx.service"
+        fi
+        #
+        [[ -d "/home/${username}/.config/Jellyfin" ]] && rm -rf "/home/${username}/.config/Jellyfin" | :
+        [[ -d "/home/${username}/.cache/jellyfin" ]] && rm -rf "/home/${username}/.cache/jellyfin" | :
+        [[ -d "/home/${username}/.aspnet" ]] && rm -rf "/home/${username}/.aspnet" | :
+    fi
+    #
+    ## legacy end
+    #
+    # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
+    wget -q -O - https://repo.jellyfin.org/debian/jellyfin_team.gpg.key | sudo apt-key add -
+    echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$(. /etc/os-release; echo $ID) $(lsb_release -cs) main" > /etc/apt/sources.list.d/jellyfin.list
+    #
+    # install jellyfin and jellyfin-ffmepg using apt functions. 
+    apt_update #forces apt refresh
+    apt_install jellyfin jellyfin-ffmpeg
+    #
+    # Configure the new jellyfin service.
+    systemctl -q stop "jellyfin.service"
+    #
+    # Add the jellyfin user to the master user's group to use our ssl certs.
+    usermod -a -G ${username} jellyfin
+    #
+    # Set the correct and required permissions of any directories we created or modified.
+    [[ -d "/home/${username}/.ssl" ]] && chown "${username}.${username}" -R "/home/${username}/.ssl" || :
+    [[ -d "/home/${username}/.ssl" ]] && chmod -R g+r "/home/${username}/.ssl" || :
+    #
+    # Set the default permissions after we have migrated our data
+    [[ -d "/etc/jellyfin" ]] && chown -R jellyfin:jellyfin "/etc/jellyfin" || :
+    [[ -f "/etc/jellyfin/logging.json" ]] && chown jellyfin:root "/etc/jellyfin/logging.json" || :
+    [[ -d "/etc/jellyfin" ]] && chown jellyfin:adm "/etc/jellyfin" || :
+    [[ -d "/var/lib/jellyfin" ]] && chown -R jellyfin:jellyfin "/var/lib/jellyfin" || :
+    [[ -d "/var/lib/jellyfin" ]] && chown jellyfin:adm "/var/lib/jellyfin" || :
```

Comment:
Why are all these in the conditions? Shouldn't they be guaranteed to be installed if the `apt_install` was ran? if `apt_install` fails, the script will be killed. Might be good to move the `apt_install` as high as possible so people don't possibly end up in some in-between state

Reply:
also I don't think I have an `adm` on my box, is this a debian thing?

Reply:
the package does this. i tested multiple times

adm: Group adm is used for system monitoring tasks. Members of this group can read many log files in /var/log, and can use xconsole. Historically, /var/log was /usr/adm (and later /var/adm), thus the name of the group.

Reply:
> the package does this. i tested multiple times

Is that a reply to why the tests aren't needed?

Reply:
I removed all the `|| :` tests

Reply:
I was referring to the `[[ -d ... ]]` checks. I think all of these should exist, and if not then there's a problem that should be thrown at the user

Reply:
done, removed.

Reply:
/resolved